autoscaling.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_launch_configuration" "asg_launch" {
+  name_prefix   = "asg-launch"
+  image_id      = "ami-068257025f72f470d"
+  instance_type = "t3.small"
+  # user_data       = file("user_data.sh")
```

Comment:
Why is this line commented ? If there's no need for it, it should be removed completely.

Reply:
we have not merged that user_data file here, but we will use it..that's why.